lib/game.py

```diff
@@ -22,7 +23,7 @@ def __init__(self,rules=Rules(),grid=Grid(),period_retention=0):
   def random_grid(self,reset_seed=True):
     for x in range(self.grid.x):
       for y in range(self.grid.y):
-        self.grid.state[x,y] = random.choice(range(self.rules.amount))
+        self.grid.state[x][y] = random.choice(range(self.rules.amount))
```

Comment:
I was under the impression the other way was more in keeping with numpy ndarrays?

Reply:
mate it's your change lol. Just got incorporated into my code somehow because of a commit that was made while I was making this one.